### PR TITLE
Fix: 503 bug

### PR DESCRIPTION
**Pull request by menachem554**

### CODE REVIEW

**Review by yarinBenisty — APPROVED** [review]
Good job!

**Review by maya-fisher — APPROVED** [review]

File: src/views/errors/503.vue [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
 <script>
 import { isAlive } from "@/api/healthcheck";
+import router from '../../router/index';
```
Comment (yoavrozner) [inline src/views/errors/503.vue:37]: 1. Use the `@` for smarter folder navigation.
2. Take a look on the vue syntax `this.$router`.
3. Default imports comes before the non default import and not after.

**Review by yoavrozner — CHANGES_REQUESTED** [review]
Well done @menachem554 !
Please see my notes.

**Review by yoavrozner — APPROVED** [review]
Well done!

**Review by yarinBenisty — APPROVED** [review]

**Pull request merged**